tests/bench/acp/acp_local_test.go

```diff
@@ -8,6 +8,8 @@
 // by the Apache License, Version 2.0, included in the file
 // licenses/APL.txt.
 
+//go:build !js
```

Comment:
question: This is a bit annoying to have to include, why is it required?

Reply:
I think these can be removed now. I'll double check.

Reply:
It looks like they are not needed. I've removed all of the build flags from `tests/bench`.

Reply:
Nice! Thanks Keenan